components/ui/Footer.js

```diff
@@ -15,41 +18,34 @@ export const Footer = () => {
         </p>
         <p>&copy; 2024 Galactica. All rights reserved.</p>
       </div>
-      {/* TASK - React 1 week 2 */}
-      {/* Create a new List for the Pages */}
-      {/* We need to use the <Link /> component here */}
-      {/* <div className={styles.pages}>
+      <div className={styles.pages}>
         <h3>Pages</h3>
         <ul>
-          <li> <Link/> </li>
-          ...
-        </ul>
-      </div> */}
-      {/* Docs for the Link: https://nextjs.org/docs/pages/api-reference/components/link */}
-
-      <div className={styles.footerLinks}>
-        <h3>Follow us</h3>
-        <ul className={styles.footerList}>
-          <li>
-            <a href="https://linkedin.com">LinkedIn</a>
-          </li>
           <li>
-            <a href="https://facebook.com">Facebook</a>
+            <Link href="/">Home</Link>
           </li>
           <li>
-            <a href="https://instagram.com">Instagram</a>
+            <Link href="/about_us">About Us</Link>
           </li>
           <li>
-            <a href="https://tiktok.com">Tiktok</a>
+            <Link href="/destination">Destination</Link>
           </li>
           <li>
-            <a href="https://google.com">On the streets at night</a>
+            <Link href="/nasa_collaboration">NASA Collaboration</Link>
           </li>
-          {/* TASK - React 1 week 2 */}
-          {/* Create a <SocialMediaItem /> component and replace all of the list items! */}
-          {/* it should accept the following props */}
-          {/* url, title, icon */}
-          {/* For the icons, you can download 1-2 social media icons for testing and put it in the /public/socialmedia/ folder */}
+        </ul>
+      </div>
+      <div className={styles.footerLinks}>
+        <h3>Follow us</h3>
+        <ul className={styles.footerList}>
+          {socialLinks.map((link) => (
+            <SocialMediaItem
+              key={link.id}
```

Comment:
We already used the `key={index}` in many other places, and this can cause some errors. So to make it unique it could be:
(since these ids are working like the indexes :) )

```
key={`social-${index}`}
```